web/templates/settings/default_streams_list_admin.hbs

```diff
@@ -5,7 +5,7 @@
         <h3>{{t "Default channels"}}</h3>
         <div class="add_default_streams_button_container">
             {{#if is_admin}}
-                <button type="submit" id="show-add-default-streams-modal" class="button rounded sea-green">{{t "Add channel" }}</button>
+                <button type="submit" id="show-add-default-streams-modal" class="button new-style rounded sea-green">{{t "Add channel" }}</button>
```

Comment:
I don't think adding `new-style` class here is useful here, since it does not apply any CSS rules to the button element. CSS rules are written for `new-style` class being used on a parent.

Reply:
Yep, removed!